api-server/api-server-common/src/storage/impls/in_memory/transactional/read.rs

```diff
@@ -48,6 +48,13 @@ impl<'t> ApiServerStorageRead for ApiServerInMemoryStorageTransactionalRo<'t> {
         self.transaction.get_block(block_id)
     }
 
+    async fn get_transaction_with_block(
```

Comment:
Why do we need a function to return a transaction with a block?

Reply:
It is with the block aux data, used to get the block timestamp and height for the transaction endpoint's confirmations and timestamp.